spec/support/coronavirus_feature_steps.rb

```diff
@@ -451,7 +451,7 @@ def stub_discard_coronavirus_page_no_draft
   end
 
   def i_see_subsection_one_in_position_one
-    expect(find(".app-c-reorderable-list:first-child")).to have_content "I am first"
```

Comment:
Can this commit be part of the previous one where tests are already being fixed?

Reply:
this was meant to be fixed up - let me do it now